src/feature/control/control.c

```diff
@@ -276,25 +276,36 @@ peek_connection_has_http_command(connection_t *conn)
 }
 
 /**
- * Helper: take a nul-terminated command of given length, and find where
- * the command starts and the argument begins.  Separate them with a NUL,
- * and return a pointer to the arguments.
+ * Helper: take a nul-terminated command of given length, and find where the
+ * command starts and the argument begins.  Separate them; allocate a new
```

Comment:
s/argument begins/arguments begin/, probably?

Reply:
Fixup in cd994ceb6ce9b2.  Also fixes an "and and" later in the comment.

Reply:
Fixup in cd994ceb6ce9b2.  Also fixes an "and and" later in the comment.